spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/cql/Schema.scala

```diff
@@ -151,11 +151,12 @@ case class TableDef(
     regularColumns: Seq[ColumnDef],
     indexes: Seq[IndexDef] = Seq.empty,
     isView: Boolean = false,
-    options: String = "") extends StructDef {
+    options: Seq[String] = Seq.empty) extends StructDef {
```

Comment:
Perhaps this should be tableOptions (matching the cql docs), and also now that I think about this perhaps it fits a Map better than a sequence? This would make it much clearer that we are looking for a set of key-value pairs.